policyengine_us/variables/gov/ssa/ssi/eligibility/income/deemed/from_ineligible_spouse/ssi_spousal_deeming_applies.py

```diff
@@ -0,0 +1,42 @@
+from policyengine_us.model_api import *
+
+
+class ssi_spousal_deeming_applies(Variable):
+    value_type = bool
+    entity = Person
+    label = "SSI spousal deeming applies"
+    definition_period = YEAR
+    reference = "https://www.law.cornell.edu/cfr/text/20/416.1163"
+    documentation = """
+    Returns True when spousal deeming applies according to 20 CFR §416.1163.
+
+    Spousal deeming applies when:
+    1. Person is an eligible individual (not part of an eligible couple)
+    2. Spouse is ineligible (not aged/blind/disabled)
+    3. Spouse's leftover income (after child allocations) exceeds the difference
+       between couple FBR and individual FBR
+
+    When deeming applies, the benefit calculation uses the couple FBR instead
+    of the individual FBR, recognizing that two people have higher expenses.
+    """
+
+    def formula(person, period, parameters):
+        # Only applies to eligible individuals (not eligible spouses in a couple)
+        is_eligible_individual = person("is_ssi_eligible_individual", period)
+
+        # Get spouse's leftover income (after child allocations)
+        spouse_earned = person(
```

Comment:
```suggestion
        spouse_deemed_earned_income = person(
```

Reply:
Can we be a bit descriptive in the variable names - just to make sure we accurately represent everything?